README.md

```diff
@@ -2037,6 +2038,28 @@ There are two placeholders that can be used in the conditions for endpoints of t
 `NOERROR`, `FORMERR`, `SERVFAIL`, `NXDOMAIN`, etc.
 
 
+### Monitoring an endpoint using MQTT
+Defining an `mqtt` configuration in an endpoint will automatically mark said endpoint as an endpoint of type MQTT:
+```yaml
+endpoints:
+  - name: example-mqtt-query
+    url: "wss://example.com/mqtt"
```

Comment:
why are you using a websocket endpoint for monitoring mqtt? 🤔 
shouldn't we be using mqtt?

Reply:
wouldn't this cause confusion, given that a websocket endpoint may be defined as a websocket endpoint internally? See the `TypeMQTT     Type = "MQTT"` change you made

Reply:
Websocket is a valid transport for MQTT: https://docs.oasis-open.org/mqtt/mqtt/v3.1.1/os/mqtt-v3.1.1-os.html#_Toc398718127

Reply:
Sounds good to me.